roboteq_driver/src/CMakeLists.txt

```diff
@@ -3,6 +3,7 @@ add_executable(${PROJECT_NAME}_node driver controller channel)
 add_dependencies(${PROJECT_NAME}_node ${PROJECT_NAME}_script roboteq_msgs_generate_messages_cpp)
 target_link_libraries(${PROJECT_NAME}_node ${PROJECT_NAME}_script ${catkin_LIBRARIES})
 set_target_properties(${PROJECT_NAME}_node PROPERTIES OUTPUT_NAME driver_node PREFIX "")
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wall -Wextra -Wno-unused-parameter")
```

Comment:
Would prefer `set_directory_properties`, see:

https://github.com/ros/ros_comm/search?q=set_directory_properties&type=Code

Otherwise this all LGTM.